src/table/table-body.jsx

```diff
@@ -384,16 +384,16 @@ const TableBody = React.createClass({
 
   _onCellClick(e, rowNumber, columnNumber) {
     e.stopPropagation();
-    if (this.props.onCellClick) this.props.onCellClick(rowNumber, this._getColumnId(columnNumber));
+    if (this.props.onCellClick) this.props.onCellClick(rowNumber, this._getColumnId(columnNumber), e);
```

Comment:
Hi, thanks for reply.

`event` instead of `e` sounds better you are right, but what I have seen in this component author used just `e` as variable name so I wanted to keep with this idea.

Moreover I agree that moving this param on the first place can break something so that's why I pushed it as the last item ;)

Personally it helps me so much in order to make some event delegations :)

Cheers!